sample/ApiHubTrigger-Batch/function.json

```diff
@@ -0,0 +1,17 @@
+{
+  "bindings": [
+    {
+      "type": "apiHubTrigger",
+      "connectionString": "%dropbox%",
```

Comment:
Don't use "connectionString" - see all the other bindings. We've standardized on "connection". Also, the value shouldn't have %% - it should be a raw app setting name